engine/src/flutter/fml/task_runner.cc

```diff
@@ -63,6 +64,16 @@ void TaskRunner::RunNowOrPostTask(const fml::RefPtr<fml::TaskRunner>& runner,
   }
 }
 
+void TaskRunner::PostTaskSync(const fml::RefPtr<fml::TaskRunner>& task_runner,
```

Comment:
Can you add a FML_DCHECK that we aren't running on the task_runner to avoid some deadlock?

Reply:
done